test/fixtures/tradeQuote.ts

```diff
@@ -201,7 +201,7 @@ export const tradeQuoteFixtures = {
     from: '0xd7dc13984d4fe87f389e50067fb3eedb3f704ea0',
     fromTokenAddress: '0x2791bca1f2de4661ed88a30c99a7a9449aa84174',
     toTokenAddress: '0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6',
-    exchangeAdapterName: 'ZeroExApiAdapterV4',
+    exchangeAdapterName: 'ZeroExApiAdapterV5',
```

Comment:
Could we import this from `tradeQuoter` utils?

Reply:
Yes, smart.